op-challenger/fault/caller.go

```diff
@@ -53,7 +57,7 @@ func (fc *FaultCaller) LogGameInfo(ctx context.Context) {
 		fc.log.Error("failed to get claim count", "err", err)
 		return
 	}
-	fc.log.Info("Game info", "claims", claimLen, "status", GameStatusString(status))
+	fc.log.Info("Game info", "claims", claimLen, "status", status.String())
```

Comment:
We don't actually need to call `.String()` here right - it should just happen automatically?

Reply:
Same for the other places its used in logging.